build-aux/snap/snapcraft.yaml

```diff
@@ -1,9 +1,9 @@
 name: tuba
-base: core22
+base: core24
 version: git
 adopt-info: tuba
 
-grade: devel
+grade: stable
```

Comment:
Is this needed?

Reply:
devel snaps can't be pushed to stable to candidate channels

Reply:
This is only for devel purposes though, the release config lives at https://github.com/GeopJr/tuba-snap/

Reply:
We can directly publish this snap to stable from here. As the snap is far behind right now.

Reply:
Added workflow to publish this to store.